src/components/AppContent/AppContent.vue

```diff
@@ -120,6 +120,16 @@ import isMobile from '../../mixins/isMobile'
 
 const browserStorage = getBuilder('nextcloud').persist().build()
 
+let hasGlobalAppName = true
+let globalAppName
+try {
+	// This will throw a ReferenceError when the global variable is missing
+	// In that case we have to make the pane config key required
+	globalAppName = appName
```

Comment:
usually doing a `if (window.appName)` should work for checking for global var existence

Reply:
I thought that was only valid in a browser context, I'm not sure how webpack usually defines globals. Are they pushed to `window` @skjnldsv ?

Reply:
No, we don't want them on window